lib/db-utils.js

```diff
@@ -207,12 +214,19 @@ module.exports = (srf, logger) => {
           const [a] = await pp.query(sqlAccountBySid, [[matches[0].account_sid]]);
           if (0 === a.length) return failure;
           logger.debug({matches}, `found registration carrier using ${uri.host} and ${uri.user}`);
+          const sql =
+            `SELECT * FROM phone_numbers WHERE number = '${did}'
```

Comment:
could we just make this `SELECT application_sid` instead of `SELECT *`?  And same above